bundles/AdminBundle/Controller/Admin/Asset/AssetController.php

```diff
@@ -632,7 +632,7 @@ public function addFolderAction(Request $request)
      */
     public function deleteAction(Request $request)
     {
-        if ($request->get('type') == 'childs') {
+        if ($request->get('type') === 'children') {
```

Comment:
I like this! 👍 
But it looks like it might be a minor BC break, right? What if a custom bundle/integration calls `/admin/asset/delete` with `type=childs`. It's mostly the usage of the getters/setters inside Pimcore I was commenting about.

Reply:
Yes I was searching for all childs typing. I will revert this.

Reply:
Maybe both can be supported, while deprecating the `childs` variant, and removing it in `v11`?

Reply:
Yes I add both options and trigger an deprecation warning when childs is used